Library/Homebrew/tap.rb

```diff
@@ -284,8 +284,9 @@ def install(options = {})
 
     link_completions_and_manpages
 
-    formula_count = formula_files.size
-    puts "Tapped #{Formatter.pluralize(formula_count, "formula")} (#{path.abv})" unless quiet
+    casks = Formatter.pluralize(cask_files.count, "cask")
+    formulae = Formatter.pluralize(formula_files.count, "formula")
+    puts "Tapped #{formulae} and #{casks} (#{path.abv})." unless quiet
```

Comment:
How about not outputting formulae or casks if they are zero? Just an idea (and I can be talked out of it)

Reply:
I like this idea. How about adding a command count, too? Like:

```
Tapped 3 casks.
Tapped 14 casks and 2 commands.
Untapped 2349 formulae.
Untapped 2673 formulae and 5 commands.
```

Reply:
Yeh, that would be good 👍